packages/contracts/test/voting/multisig.ts

```diff
@@ -502,6 +617,42 @@ describe('Multisig', function () {
       it('returns `true` if the approver is listed', async () => {
         expect(await multisig.canApprove(id, signers[0].address)).to.be.true;
       });
+
+      it("returns `false` if the proposal hasn't started yet", async () => {
+        await multisig.createProposal(
+          dummyMetadata,
+          dummyActions,
+          false,
+          false,
+          await timestampIn(2000),
+          await timestampIn(5000)
+        );
+
+        expect(await multisig.canApprove(1, signers[0].address)).to.be.false;
+
+        await ethers.provider.send('evm_increaseTime', [2000]);
+        await ethers.provider.send('evm_mine');
```

Comment:
You could use `advanceTimeTo` from `voting.ts`. Fine if you don't

Reply:
done